test/e2e/run_networking_test.go

```diff
@@ -1100,27 +1100,33 @@ EXPOSE 2004-2005/tcp`, ALPINE)
 		pod := "testpod"
 		session := podmanTest.Podman([]string{"pod", "create", "--name", pod})
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(0))
+		Expect(session).Should(ExitCleanly())
 
 		net := createNetworkName("IntTest")
 		session = podmanTest.Podman([]string{"network", "create", net})
 		session.WaitWithDefaultTimeout()
 		defer podmanTest.removeNetwork(net)
-		Expect(session).Should(Exit(0))
+		Expect(session).Should(ExitCleanly())
 
 		pod2 := "testpod2"
 		hostname := "hostn1"
 		session = podmanTest.Podman([]string{"pod", "create", "--network", net, "--name", pod2, "--hostname", hostname})
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(0))
+		Expect(session).Should(ExitCleanly())
 
 		session = podmanTest.Podman([]string{"run", "--name", "con1", "--network", net, ALPINE, "nslookup", "con1"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
+		if !Containerized() {
+			Expect(session.ErrorToString()).To(Equal("nslookup: can't resolve '(null)': Name does not resolve"))
+		}
```

Comment:
This is not expected, if I remember correctly string parsing nslookup output is not a good idea in general as this format can change. When I test with the official docker.io/library/alpine image it will not print this line, so I assume they fixed whatever problem caused these lines.

So my question: can we update quay.io/libpod/alpine to something newer which does not print these bogus lines? And ideally to not overwrite the `latest` tag so we do not break existing CI.

Reply:
Thank you! You're right, that was an artifact of the alpine image. I've switched to `CITEST_IMAGE` instead (in the `run check dns` test only) and that allows using `ExitCleanly()`. Can you see if this is better?

Re: parsing command output, I agree, but nslookup is never going to change in the `testimage:20221018` image. When we bump to `testimage:20YYMMDD`, if nslookup output fails, it's trivial to change the test and that will never affect any past or future test.

Reply:
> Thank you! You're right, that was an artifact of the alpine image. I've switched to CITEST_IMAGE instead (in the run check dns test only) and that allows using ExitCleanly(). Can you see if this is better?

Sure works for me.